import { AppError } from "./AppError";

export class NotFoundError extends AppError{
    constructor(message = 'Recurso n√£o encontrado'){
        super(message, 404)
    }
}